.editorconfig

```diff
@@ -0,0 +1,41 @@
+# http://editorconfig.org
```

Comment:
@gidili @tarelli I think it is up to you to accept this file. I like it and, actually, I added this feature in [the lint branch](https://github.com/openworm/org.geppetto.frontend/blob/lint/src/main/webapp/.editorconfig) that wasn't merged into development. It brings code formatting for a lot of languages and provides compatibility with most of the IDEs (http://editorconfig.org/)... but we are not using it anywhere in Geppetto at the moment. Maybe you want to give it a try for this python project but then I would recommend you add it to django-geppetto-template as well and agree on the conventions

Reply:
@adrianq l'd say et's keep it and we can add it to the other one too!